### PR TITLE
chore(shipping): improve shipping related documentation

### PR DESCRIPTION
**Pull request by paul-vd**

### CODE REVIEW

File: source/docs/advanced/shipping/colissimo.md [hunk 1]
```diff
@@ -5,19 +5,19 @@ title: Colissimo integration
 
 This page contains information about the different ways you can accept shippings with [Colissimo](https://www.laposte.fr/colissimo) in your Front-Commerce application.
 
-## Magento2 module
+## Prerequisites
+
+No matter the backend solution, the Colissimo Front-Commerce modules requires a Map component to display pickup points. Thus, before proceeding, please make sure that you've chosen a Map implementation from [Display a map](/docs/advanced/features/display-a-map.html).
+
```
Comment (paul-vd) [inline source/docs/advanced/shipping/colissimo.md:11]: Ensure that the format of the Colissimo & Mondial-Relay docs are the same.

File: source/docs/advanced/shipping/colissimo.md [hunk 2]
```diff
@@ -5,19 +5,19 @@ title: Colissimo integration
...
 </blockquote>
 
 The integration of Colissimo in Magento 2 & Front-Commerce relies on [Magentix's module](https://colissimo.magentix.fr/magento-2/) for pickup points. If you don't need pickup points, any module should be supported by default.
 
-## Install a map module
-
-This module uses the Front-Commerce Map's component to display pickup points. Thus, before proceeding, please make sure that you've chose a Map implementation from [Display a map](/docs/advanced/features/display-a-map.html).
-
-### Installation
+Then, within your Front-Commerce project, you have to:
```
Comment (paul-vd) [inline source/docs/advanced/shipping/colissimo.md:20]: Ensure that the format of the Colissimo & Mondial-Relay docs are the same.

File: source/docs/advanced/shipping/mondial-relay.md [hunk 3]
```diff
@@ -123,6 +123,4 @@ const ComponentMap = {
 };
 ```
 
-## That's it
-
```
Comment (paul-vd) [inline source/docs/advanced/shipping/mondial-relay.md:127]: This created a title in the sidebar which is confusing.

**Review by abailleulfc — APPROVED** [review]

File: source/docs/advanced/shipping/colissimo.md [hunk 4]
```diff
@@ -5,19 +5,19 @@ title: Colissimo integration
 
 This page contains information about the different ways you can accept shippings with [Colissimo](https://www.laposte.fr/colissimo) in your Front-Commerce application.
 
-## Magento2 module
+## Prerequisites
+
+No matter the backend solution, the Colissimo Front-Commerce modules requires a Map component to display pickup points. Thus, before proceeding, please make sure that you've chosen a Map implementation from [Display a map](/docs/advanced/features/display-a-map.html).
```
Comment (mo-fc) [inline source/docs/advanced/shipping/colissimo.md:10]:
```suggestion
No matter the backend solution, the Colissimo Front-Commerce modules requires a Map component to display pickup points. Thus, before proceeding, please make sure that you have chosen a Map implementation from [Display a map](/docs/advanced/features/display-a-map.html).
```

File: source/docs/advanced/shipping/custom-shipping-information.md [hunk 5]
```diff
@@ -32,9 +32,10 @@ If your shipping method is correctly registered in your backend, the following G
 
 ## Display a custom input in your shipping method
 
-Once you've done this request, you will be able to get the `carrier_code` and the `method_code` of your shipping method. You should use those to register a new component in `theme/modules/Checkout/ShippingMethod/AdditionalShippingInformation/getAdditionalDataComponent.js` by following this template:
+Once you've done this request, you will be able to get the `carrier_code` and the `method_code` of your shipping method. You should use those to register a new component by overriding the [`getAdditionalDataComponent.js`](https://gitlab.com/front-commerce/front-commerce/-/blob/main/src/web/theme/modules/Checkout/ShippingMethod/AdditionalShippingInformation/getAdditionalDataComponent.js) by following this template:
```
Comment (mo-fc) [inline source/docs/advanced/shipping/custom-shipping-information.md:35]:
```suggestion
Once you have done this request, you will be able to get the `carrier_code` and the `method_code` of your shipping method. You should use those to register a new component by overriding the [`getAdditionalDataComponent.js`](https://gitlab.com/front-commerce/front-commerce/-/blob/main/src/web/theme/modules/Checkout/ShippingMethod/AdditionalShippingInformation/getAdditionalDataComponent.js) by following this template:
```

File: source/docs/advanced/shipping/mondial-relay.md [hunk 6]
```diff
@@ -9,7 +9,7 @@ This page contains information about the different ways you can ship orders with
 
 No matter the backend solution, the MondialRelay Front-Commerce modules requires a Map component to display pickup points. Thus, before proceeding, please make sure that you've chosen a Map implementation from [Display a map](/docs/advanced/features/display-a-map.html).
 
-## Magento2 based application
+## Integrate with Magento2
```
Comment (mo-fc) [inline source/docs/advanced/shipping/mondial-relay.md:12]: check links to #Magento2-based-application there are two in the migration guide that needs to be updated

File: source/docs/advanced/shipping/mondial-relay.md [hunk 7]
```diff
@@ -65,7 +65,7 @@ const ComponentMap = {
 };
 ```
 
-## Magento1 based application
+## Integrate with Magento1
```
Comment (mo-fc) [inline source/docs/advanced/shipping/mondial-relay.md:68]: Same here there is one link in the migration guide pointing here

**Review by mo-fc — CHANGES_REQUESTED** [review]

File: source/docs/advanced/shipping/colissimo.md (hunk 4 above)
Comment (paul-vd) [reply source/docs/advanced/shipping/colissimo.md:10]: `you've` seems grammatically correct for `you have` in this case?

File: source/docs/advanced/shipping/custom-shipping-information.md (hunk 5 above)
Comment (paul-vd) [reply source/docs/advanced/shipping/custom-shipping-information.md:35]: Same here.

File: source/docs/advanced/shipping/mondial-relay.md (hunk 6 above)
Comment (paul-vd) [reply source/docs/advanced/shipping/mondial-relay.md:12]: Thanks, yea I forgot to do that 💡

**Review by mo-fc — APPROVED** [review]

**Pull request merged**